generator/templates/rust/crate/src/lib.rs.mustache

```diff
@@ -26,6 +26,8 @@ use gax::error::{Error, HttpError};
 use google_cloud_auth::{Credential, CredentialConfig};
 use std::sync::Arc;
 
+const DEFAULT_HOST: &str = "https://{{DefaultHost}}/";
```

Comment:
This change seems unrelated.  Nice change, but maybe should have its own PR.  Ditto for the `client.go.mustache` change.